### ISSUE TITLE
service tag in pymongo span cannot be overridden

### ISSUE THREAD

**Issue by osawa-eiji** [DataDog/dd-trace-py#10897]
### Summary of problem
The following is specified as in [the documentation](https://ddtrace.readthedocs.io/en/stable/integrations.html#id130), but the service tag is not overridden and the default "pymongo" is output.
```python
    ddtrace.config.pymongo["service"] = "anther-mongo"
    ddtrace.patch(pymongo=True)
```
Incidentally, it is overwritten without any problems in redis.
```python
    ddtrace.config.redis["service"] = "anther-redis"
    ddtrace.patch(redis=True)
```

Can you think of any cause for this event, such as an incorrect designation method?

### Which version of dd-trace-py are you using?
2.14.0

### Which version of pip are you using?
Use poetry instead of pip.
poetry version is 1.8.3

### Which libraries and their versions are you using?
pymongo==4.9.1

### How can we reproduce your problem?
Execute with the following designation
```python
    ddtrace.config.pymongo["service"] = "anther-service"
    ddtrace.patch(pymongo=True)
```

### What is the result that you get?
service tag in pymongo span not overridden.

### What is the result that you expected?
service tag in pymongo span is overwritten.

**Comment by quinna-h** [comment]
Hi @osawa-eiji ! 

Thanks for raising this issue. This should be fixed by #10917. 

Please follow up if you have further issues.

**Comment by osawa-eiji** [comment]
Hi @quinna-h 

Thank you for your response!
We will check this when it is released.
